torch/csrc/jit/ir/irparser.cpp

```diff
@@ -306,6 +338,29 @@ void IRParser::parseAttr(Node* n) {
       default:
         throw ErrorReport(L.cur().range) << "Unexpected attr type";
     }
+  } else if (L.cur().text() == "annotate") {
+    L.next();
+    L.expect('(');
+    auto type = L.cur().text();
+    if (type != "List" && type != "Dict") {
+      throw ErrorReport(L.cur().range)
+          << "Unexpected annotation (only List and Dict can be parsed)";
+    }
+    L.next();
+    // use the profiling annotation instead of the `annotate()` type.
+    // so, we throw away the annotation value
+    // Note: we could also use script_type_parser
+    bypassTypeAnnotationList();
+    L.expect(',');
+    // expect an empty definition
```

Comment:
It wont always be https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/ivalue.cpp#L484 empty.. although we can defer handling this case

Reply:
added this info to the comment